### ISSUE TITLE
Update examples design

### ISSUE THREAD

**Issue by 9renpoto** [userdive/agent.js#382]
# TL;DR

based bootstrap from cdn.

- [ ] v4 ?
- [ ] other framework ?

### Examples Toc or links

- https://userdive/github.io/agent.js/with-angular1/index.html
- https://userdive/github.io/agent.js/index.html
- https://userdive/github.io/agent.js/linker/linker.html
- https://userdive/github.io/agent.js/with-react/index.html
- https://userdive/github.io/agent.js/simple/multiple.html
- https://userdive/github.io/agent.js/simple/simple.html
- https://userdive/github.io/agent.js/built-in/index.html
- https://userdive/github.io/agent.js/with-vue/index.html
- https://userdive/github.io/agent.js/with-google-tag-manager/index.html
- https://userdive/github.io/agent.js/with-angular/index.html

### Simple Form

input, checkbox, selector
